test/cmd/legacy-script.sh

```diff
@@ -98,10 +98,10 @@ secrets="secrets"
 serviceaccounts="serviceaccounts"
 services="services"
 statefulsets="statefulsets"
-static="static"
+export static="static"
```

Comment:
this is a pretty goofy thing to export, what is reading this?

Reply:
Yeah, I got it. I've removed these kind of external use in last commit.